package.json

```diff
@@ -277,7 +277,7 @@
         "sass-lint": "^1.13.1",
         "sass-loader": "^8.0.2",
         "scroll-into-view-if-needed": "^2.2.20",
-        "semantic-release": "^16.0.2",
+        "semantic-release": "^24.2.3",
```

Comment:
do we know what the breaking changes are between versions? will this affect our releases?

Reply:
updated description